nlu/docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: '3.9'
+services:
+  mlflow_postgres:
+    image: bitnami/postgresql
+    container_name: postgres_db
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=mlflow_db
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+
+  mlflow_server:
+    restart: always
+    build:
+      context: ./docker
+      dockerfile: Dockerfile  # Specify the Dockerfile explicitly
+    image: mlflow
+    container_name: mlflow_server
+    environment:
+      - BACKEND_STORE_URI=postgresql://postgres:postgres@mlflow_postgres:5432/mlflow_db  # Connection string to Postgres
+      - ARTIFACT_STORE_URI=./mlruns  # Local directory for storing artifacts
+    ports:
+      - "5002:5000"  # Expose MLflow UI
+    volumes:
+      - ./mlruns:/mlruns  # Mount local directory for MLflow artifacts
```

```suggestion
      - mlruns:/mlruns  # Mount local directory for MLflow artifacts
```
Add "mlruns:" to the volumes